sentry-log4j2/src/main/java/io/sentry/log4j2/SentryAppender.java

```diff
@@ -262,6 +262,7 @@ public void append(final @NotNull LogEvent eventObject) {
     sdkVersion = SdkVersion.updateSdkVersion(sdkVersion, name, version);
 
     sdkVersion.addPackage("maven:io.sentry:sentry-log4j2", version);
+    sdkVersion.addIntegration("logback");
```

Comment:
this should be `log4j2` right? Also, can we uppercase the first letter to align with others?

Reply:
```suggestion
    sdkVersion.addIntegration("Log4j2");
```